libraries/AP_Math/crc.cpp

```diff
@@ -374,6 +374,53 @@ uint16_t crc16_ccitt(const uint8_t *buf, uint32_t len, uint16_t crc)
     return crc;
 }
 
+/* right shift table */
+static const uint16_t crc16tab_r[256] = {
```

Comment:
Which CCITT is this? https://crccalc.com/ There's a 99.9% chance it's already implemented here